features/all-positions/AllLiquidations.tsx

```diff
@@ -211,113 +242,144 @@ const AllLiquidations = () => {
-                    {translateLiquidationStatus(
-                      Number(liquidation.liquidationTimestamp),
-                      liquidation.status
-                    )}
+                    <strong>
+                      <SortableTableColumnHeader
+                        sortField={SORT_FIELD.MAX_DISPUTABLE_PRICE}
+                      >
+                        {invertDisputablePrice ? `Minimum` : `Maximum`}{" "}
+                        Disputable Price{" "}
+                      </SortableTableColumnHeader>
+                    </strong>
                   </TableCell>
-                </TableRow>
-              );
-            })}
-          </TableBody>
-        </Table>
-        <PageButtons>
-          <div
-            onClick={() => {
-              setPage(page === 1 ? page : page - 1);
-            }}
-          >
-            <Arrow faded={page === 1 ? true : false}>←</Arrow>
-          </div>
-          {"Page " + page + " of " + maxPage}
-          <div
-            onClick={() => {
-              setPage(page === maxPage ? page : page + 1);
-            }}
-          >
-            <Arrow faded={page === maxPage ? true : false}>→</Arrow>
-          </div>
-        </PageButtons>
-      </TableContainer>
+                </Tooltip>
+
+                <TableCell align="right"></TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {reformattedLiquidationData.map((sponsor: string) => {
+                const liquidation = liquidations[sponsor];
+                return (
+                  <StyledTableRow key={sponsor}>
+                    <TableCell component="th" scope="row">
+                      <a href={getEtherscanUrl(sponsor)} target="_blank">
+                        {prettyAddress(sponsor)}
+                      </a>
+                    </TableCell>
+                    <TableCell align="left">
+                      {prettyDate(Number(liquidation.liquidationTimestamp))}
+                    </TableCell>
+                    <TableCell align="right">
+                      {translateLiquidationStatus(
+                        Number(liquidation.liquidationTimestamp),
+                        liquidation.status
+                      )}
+                    </TableCell>
+                    <TableCell align="right">
+                      {prettyBalance(Number(liquidation.lockedCollateral))}
+                    </TableCell>
+                    <TableCell align="right">
+                      {prettyBalance(Number(liquidation.tokensLiquidated))}
+                    </TableCell>
+                    <TableCell align="right">
+                      {prettyBalance(
+                        getDisputablePrice(liquidation.maxDisputablePrice)
+                      )}
+                    </TableCell>
+
+                    <TableCell align="right">
+                      <MoreInfo
+                        onClick={() => handleOpenActionsDialog(sponsor)}
```

Comment:
After merging in `master`, you'll likely have to pass in `liquidation.sponsor` instead of `sponsor`. And the `key` for the `.map()` will be `sponsorPlusId` and you can access `liquidation = liquidation[sponsorPlusId]`